tests/expectations/test_expectation.py

```diff
@@ -168,7 +169,7 @@ def test_expectation_succeeds_with_valid_mostly(fake_expectation_cls, config):
 def test_multicolumn_expectation_validation_errors_with_bad_mostly(
     fake_expectation_cls, config
 ):
-    with pytest.raises(InvalidExpectationConfigurationError):
+    with pytest.raises(pydantic.ValidationError):
```

Comment:
A breaking change but we should leverage Pydantic's type instead of our own configuration error if we're going to be using models